velox/exec/tests/OrderByTest.cpp

```diff
@@ -232,16 +232,6 @@ class OrderByTest : public OperatorTestBase {
     }
   }
 
-  static void reclaimAndRestoreCapacity(
```

Comment:
Can you also remove reclaimAndRestoreCapacity in AggregationTest.cpp and HashJoinTest as well? Thanks!